src/commands/synthetics/run-tests-lib.ts

```diff
@@ -300,5 +304,33 @@ export const execute = async (
 
   reportExitLogs(mainReporter, localConfig, {results})
 
-  return toExitCode(getExitReason(localConfig, {results}))
+  const exitCode = toExitCode(getExitReason(localConfig, {results}))
+
+  return {
+    results,
+    summary,
+    exitCode,
+  }
+}
+
+export const execute = async (
+  runConfig: WrapperConfig,
+  {
+    jUnitReport,
+    reporters,
+    runId,
+    suites,
+  }: {
+    jUnitReport?: string
+    reporters?: (SupportedReporter | Reporter)[]
+    runId?: string
+    suites?: Suite[]
+  }
```

Comment:
Should I extract the object type into a something like `type ExecuteOptions = `, or we're fine leaving it as-is?

Reply:
Yes, and please define that type in this file without exporting it to avoid making the API surface even bigger. It's only to factor the type

Reply:
@Drarig29 done